### PR TITLE
Neka-change avatar style path

### PR DESCRIPTION
**Pull request by istoga**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [X] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
Correct avatar styling path

## Related Tickets & Documents
- Closes # 74
  
## QA Instructions, Screenshots, Recordings
In attempting to utilize the custom avatar, found error. Styling path needed to be corrected.
No screenshot required, only error given.
  
## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [X] No, and here is why: no testing being implemented

### CODE REVIEW

**Comment by TaylorDJones11** [comment]
It looks like someone didn't pull down from the main repo before pushing and their PR was merged. I corrected this file and deleted the images folder but both are now back in the main repo. But should be an easy fix!

**Comment by istoga** [comment]
Ok @TaylorDJones11 thank you!

**Review by msarit — APPROVED** [review]

**Comment by msarit** [comment]
Hello all, I merged this straightforward change that @istoga made without the 2 approvals; so that the change can be available quickly to all open PRs. 🙏🏾

**Pull request merged**
